deploy/FlapperInit.sol

```diff
@@ -135,26 +125,39 @@ library FlapperInit {
     }
 
     function initSplitter(        
-        DssInstance memory dss,
-        address splitter,
-        SplitterConfig memory cfg
+        DssInstance      memory dss,
+        SplitterInstance memory splitterInstance,
+        SplitterConfig   memory cfg
     ) internal {
+        SplitterLike    splitter = SplitterLike(splitterInstance.splitter);
+        SplitterMomLike mom      = SplitterMomLike(splitterInstance.mom);
+
         // Sanity checks
-        SplitterLike splitter_ = SplitterLike(splitter);
-        require(splitter_.vat()     == address(dss.vat),     "Splitter vat mismatch");
-        require(splitter_.daiJoin() == address(dss.daiJoin), "Splitter daiJoin mismatch");
+        require(splitter.vat()     == address(dss.vat),     "Splitter vat mismatch");
+        require(splitter.daiJoin() == address(dss.daiJoin), "Splitter daiJoin mismatch");
+        require(splitter.farm()    == cfg.farm,             "Splitter farm mismatch");
 
-        require(cfg.burn <= WAD, "Splitter burn too high");
+        require(cfg.hump > 0,         "hump too low");
+        require(cfg.hop >= 5 minutes, "hop too low");
+        require(cfg.burn <= WAD,      "burn too high");
+
+        splitter.file("hop",  cfg.hop);
+        splitter.file("burn", cfg.burn);
+        splitter.rely(address(mom));
 
-        splitter_.file("burn", cfg.burn);
-        FarmLike farm = FarmLike(splitter_.farm());
-        farm.setRewardsDistribution(splitter);
+        FarmLike farm = FarmLike(cfg.farm);
+        farm.setRewardsDistribution(splitterInstance.splitter);
         farm.setRewardsDuration(cfg.rewardsDuration);
 
-        // Wire splitter with vow
-        splitter_.rely(address(dss.vow));
-        dss.vow.file("flapper", splitter);
+        splitter.rely(address(dss.vow));
+        dss.vow.file("flapper", splitterInstance.splitter);
+
+        dss.vow.file("hump", cfg.hump);
+        dss.vow.file("bump", cfg.bump);
+
+        mom.setAuthority(dss.chainlog.getAddress("MCD_ADM"));
 
-        dss.chainlog.setAddress("MCD_FLAP", splitter);
+        dss.chainlog.setAddress(cfg.chainlogKey, splitterInstance.splitter);
+        dss.chainlog.setAddress("SPLITTER_MOM", address(mom));
```

Comment:
We should probably also remove the current flapper mom from the chainlog.

Reply:
Well it replaces the flapper mom, so can be either here or in the flapper init.
I think it's a bit more convenient here since I assume the splitter init will only be called once, while the flappers can be alternated. Alternatively, can add it to the flapper init as an optional operation (like suggested for MCD_FLAP")

I think the pattern should be that we try to include everything we need for the first on-boarding spell, while supporting later changes is more of a nice to have thing. In this case it is part of the initial spell.

About `MCD_FLAP", yes, there is a separate comment below.

Reply:
Ok, added the cleanup to `initSplitter` in 79a0021c.